tutorials/navigation/navigation_using_agent_avoidance.rst

```diff
@@ -0,0 +1,79 @@
+.. _doc_navigation_using_agent_avoidance:
+
+Using Agent Avoidance
+=====================
+
+.. warning::
+
+    This chapter is a stub for linking and unfinished.
+
+This section is about how to use agent avoidance with the NavigationServer and 
+about technical detail how agent avoidance is implemented in Godot.
+
+For avoidance use with NavigationAgents Nodes see :ref:`doc_navigation_using_navigationagents`.
+
+Agent avoidance helps to prevent direct collision with other agents or moving obstacles
+while the agents still follow their original velocity as best as possible.
+
+
+Avoidance in Godot is implemented with the help of the RVO library (Reciprocal Velocity Obstacle).
```

Comment:
Non-specialist question: Isn't `Reciprocal Velocity Obstacle` more like the approach/algorithm, and we use **a** RVO library? Or is it really **the** RVO lib?

Reply:
Quote from their page:

> The main difference between RVO2 Library and RVO Library 1.x is the local collision avoidance technique used. RVO2 Library uses [Optimal Reciprocal Collision Avoidance](http://gamma.cs.unc.edu/CA/) (ORCA), whereas RVO Library 1.x uses [Reciprocal Velocity Obstacles](http://gamma.cs.unc.edu/RVO/) (RVO). For legacy reasons, and since both techniques are based on the same principles of reciprocal collision avoidance and relative velocity, we did not change the name of the library.